arm-sql/2014-04-01/examples/DatabaseCreateMax.json

```diff
@@ -0,0 +1,105 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "sqlcrudtest-4799",
+    "serverName": "sqlcrudtest-6440",
+    "databaseName": "testdb",
+    "api-version": "2014-04-01",
+    "parameters": {
+    "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-8931/providers/Microsoft.Sql/servers/sqlcrudtest-1932/databases/testdb",
+    "name": "testdb",
+    "type": "Microsoft.Sql/servers/databases",
+    "location": "Japan East",
+    "kind": "v12.0,user",
+      "properties": {
+          "databaseId": "816c5f7e-0e36-4eec-9c51-eee7a276c14c",
+          "edition": "Standard",
+          "status": "Online",
+          "createMode": "Default",
+          "serviceLevelObjective": "S0",
+          "collation": "SQL_Latin1_General_CP1_CI_AS",
+          "maxSizeBytes": "268435456000",
+          "creationDate": "2017-02-10T01:37:18.847Z",
+          "currentServiceObjectiveId": "f1173c43-91bd-4aaa-973c-54e79e15235b",
+          "requestedServiceObjectiveId": "f1173c43-91bd-4aaa-973c-54e79e15235b",
+          "requestedServiceObjectiveName": "S0",
+          "sampleName": null,
```

Comment:
Is there a reason why you included this property with null value in the request body? The property doesn't seem to be required, and if that's correct and you remove the null properties from the request body, the errors REQUEST_VALIDATION_ERROR reported by the tool should go away: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/204330240#L2507